django/template/engine.py

```diff
@@ -256,3 +260,53 @@ def make_origin(self, display_name, loader, name, dirs):
             return LoaderOrigin(display_name, loader, name, dirs)
         else:
             return None
+
+    def get_template_debug(self, exception, template, token):
```

Comment:
This method could use some comments. As someone not familiar with the code, the variable names don't seem to be self-explanatory.